snapshot/lib/snapshot/test_command_generator_base.rb

```diff
@@ -80,6 +81,20 @@ def derived_data_path
         Snapshot.cache[:derived_data_path] ||= (Snapshot.config[:derived_data_path] || Dir.mktmpdir("snapshot_derived"))
       end
 
+      def result_bundle_path(language, locale)
+        Snapshot.cache[:result_bundle_path] = {}
+        language_key = locale || language
+
+        unless Snapshot.cache[:result_bundle_path][language_key]
```

Comment:
This is never going to be true because on line 85 you always reset it.

Reply:
This could totally be true if someone is running multiple `snapshot`s for the same language during the same fastlane execution 😜

Reply:
😨